.drone.starlark

```diff
@@ -929,6 +929,12 @@ def notify():
 						'from_secret': config['rocketchat']['from_secret']
 					},
 					'channel': config['rocketchat']['channel']
+				},
+				'when': {
+					'status': [
+						'success',
```

Comment:
Are you sure you want to notify success AND failure? Not only failure? You should also mtch the ref `refs/heads/master`. As this step is the only one within the pipeline you could also directly move it to the trigger section below.

Reply:
We are currently notifying success, which was done from the first `.drone.starlark` in the `activity` app. Somehow that "decision" was made, and at the moment it seems reasonable because we get a list of positive green messages each morning in the builds channel (rather than silence).

Reply:
Ok to me, just wanted to confirm that :)
Than I would say move it to `trigger` and it should be fine.